index.html

```diff
@@ -2758,10 +2758,21 @@ <h3>Terms of Use</h3>
 {
   "@context": [
     "https://www.w3.org/2018/credentials/v1",
-    "https://www.w3.org/2018/credentials/examples/v1"
+    "https://www.w3.org/2018/credentials/examples/v1",
+    {
+        "VerifiablePresentationExtension": {
```

Comment:
```suggestion
        "VerifiablePresentationTermsOfUseExtension": {
```

Reply:
And add `"@protected": true` at the top of this embedded context too.